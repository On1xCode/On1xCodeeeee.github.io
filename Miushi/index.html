<!DOCTYPE html>
<html lang="ru">

<head>
	<meta charset="utf-8">
	<title>Miushi</title>
	<meta name="description" content="Your description here">
	<meta name="viewport" content="width=device-width">
	<link rel="icon" type="image/png" href="img/favicons/favicon.png">
	<link rel="apple-touch-icon" href="img/favicons/apple-touch-favicon.png">
	<meta property="og:image" content="img/@1x/preview.jpg">
	<link rel="stylesheet" href="css/style.min.css">
</head>

<body>

	<header class="header">
		<div class="header__inner">
			<div class="header__inner-top">
				<div class="top-header container">
					<div class="burger">
						<span></span>
					</div>
					<div class="top-header__logo">
						<a href="index.html">
							<object class="desktop-logo" type="image/svg+xml" data="img/header/header-logo.svg"></object>
							<object class="mobile-logo" type="image/svg+xml" data="img/header/header-mobile-logo.svg"></object>
						</a>
					</div>
					<div class="top-header__text top-header__text-adress">
						<a href="#">г. Курск, ул. Ленина, д. 3</a>
					</div>
					<div class="top-header__text top-header__text-email">
						<a href="mailto:78003332222@yandex.ru">78003332222@yandex.ru</a>
					</div>
					<div class="top-header__info info">
						<a href="tel:78003332222" class="info__number">+7 (800) 333 2 222</a>
						<span class="info__work-time">Работаем с 11.00 до 23.00</span>
					</div>
					<a href="#" class="top-header__recall recall-bnt" id="recall__open-btn">Обратный звонок</a>
				</div>
			</div>
			<div class="header__inner-bottom">
				<div class="bottom-header container">
					<nav class="bottom-header__menu">
						<ul class="menu">
							<li class="menu__item"><a href="#" class="menu__link menu__link-arrow">Меню</a></li>
							<li class="menu__item"><a href="#" class="menu__link">Акции и скидки</a></li>
							<li class="menu__item"><a href="#" class="menu__link">Доставка и оплата</a></li>
							<li class="menu__item"><a href="#" class="menu__link">Новости</a></li>
							<li class="menu__item"><a href="#" class="menu__link">Контакты</a></li>
						</ul>
					</nav>
					<div class="bottom-header__user user">
						<div class="user__option cabinet">
							<span class="user__option-ic">Кабинет</span>
						</div>
						<div class="user__option cart" id="popup__open-btn">
							<span class="user__option-ic">Корзина</span>
						</div>
					</div>
				</div>
			</div>
		</div>
	</header>

	<div class="popup hidden">
		<div class="popup__background hidden"></div>
		<div class="popup__recall recall hidden">
			<div class="recall__close-btn popup__close-btn" id="recall__close-btn"><span></span></div>
			<h3 class="recall__title popup__title">Введите ваши данные и мы вам перезвоним в течении 5-ти минут!</h3>
			<form action="POST" class="popup__form">
				<div class="form-group">
					<label for="name">Ваше имя:</label>
					<input type="text" name="name" id="recall-name">
				</div>
				<div class="form-group">
					<label for="phone">Ваш номер телефона:</label>
					<input type="phone" name="phone" id="recall-phone">
				</div>
				<p class="popup__form-error"></p>
				<button type="button" class="btn popup__form-btn" id="recall-btn">Отправить</button>
			</form>
		</div>
		<div class="popup__basket basket">
			<div class="basket__close-btn popup__close-btn" id="basket__close-btn"><span></span></div>
		</div>
		<div class="popup__menu">
			<div class="menu__close-btn popup__close-btn" id="menu__close-btn"><span></span></div>
			<nav class="popup__menu-nav">
				<ul class="menu">
					<li class="menu__item"><a href="#" class="menu__link menu__link-arrow">Меню</a></li>
					<li class="menu__item"><a href="#" class="menu__link">Акции и скидки</a></li>
					<li class="menu__item"><a href="#" class="menu__link">Доставка и оплата</a></li>
					<li class="menu__item"><a href="#" class="menu__link">Новости</a></li>
					<li class="menu__item"><a href="#" class="menu__link">Контакты</a></li>
				</ul>
			</nav>
			<div class="info popup__info">
				<a href="tel:78003332222" class="info__number">+7 (800) 333 2 222</a>
				<span class="info__work-time">Работаем с 11.00 до 23.00</span>
			</div>
			<a href="#" class="recall-bnt popup__recall-btn" id="menu-recall__open-btn">Обратный звонок</a>
			<div class="top-header__text top-header__text-adress popup-pos">
				<a href="#">г. Курск, ул. Ленина, д. 3</a>
			</div>
			<div class="top-header__text top-header__text-email popup-mail">
				<a href="mailto:78003332222@yandex.ru">78003332222@yandex.ru</a>
			</div>
		</div>
	</div>

	<section class="intro">
		<div class="intro__carousel carousel">
			<div class="carousel__body">
				<div class="slide">
					<div class="container">
						<h1 class="slide__title">Счастливые <br> часы</h1>
						<p class="slide__text">с 11:00 до 16:00 в будние дни</p>
						<a href="#" class="slide__btn btn">Подробнее</a>
					</div>
				</div>
				<div class="slide">
					<div class="container">
						<h1 class="slide__title">Счастливые <br> часы</h1>
						<p class="slide__text">с 11:00 до 16:00 в будние дни</p>
						<a href="#" class="slide__btn btn">Подробнее</a>
					</div>
				</div>
				<div class="slide">
					<div class="container">
						<h1 class="slide__title">Счастливые <br> часы</h1>
						<p class="slide__text">с 11:00 до 16:00 в будние дни</p>
						<a href="#" class="slide__btn btn">Подробнее</a>
					</div>
				</div>
			</div>
			<div class="carousel__dots"></div>
			<div class="carousel__btn carousel__btn-prev"><span></span></div>
			<div class="carousel__btn carousel__btn-next"><span></span></div>
		</div>
	</section>

	<section class="sets pd">
		<div class="container">
			<h1 class="sets__title section__title"><span>Сеты</span></h1>
			<div class="products-carousel">
				<div class="products-carousel__body sets-products">
					<!-- JS output -->
				</div>
				<div class="products-carousel__control carousel-control">
					<div class="carousel-control__btn carousel-control__btn-prev">
						<span></span></div>
					<div class="carousel-control__dots"></div>
					<div class="carousel-control__btn carousel-control__btn-next">
						<span></span></div>
				</div>
			</div>
	</section>

	<section class="rolls pd">
		<div class="container">
			<h1 class="rolls__title section__title"><span>Роллы</span></h1>
			<div class="products-carousel">
				<div class="products-carousel__body rolls-products">
					<!-- JS output -->
				</div>
				<div class="products-carousel__control carousel-control">
					<div class="carousel-control__btn carousel-control__btn-prev"><span></span></div>
					<div class="carousel-control__dots"></div>
					<div class="carousel-control__btn carousel-control__btn-next"><span></span></div>
				</div>
			</div>
		</div>
	</section>

	<section class="pizza pd">
		<div class="container">
			<h1 class="pizza__title section__title"><span>Пицца </span></h1>
			<div class="products-carousel">
				<div class="products-carousel__body pizza-products">
					<!-- JS output -->

					<div class="products-carousel__control carousel-control">
						<div class="carousel-control__btn carousel-control__btn-prev"><span></span></div>
						<div class="carousel-control__dots"></div>
						<div class="carousel-control__btn carousel-control__btn-next"><span></span></div>
					</div>
				</div>
			</div>
	</section>

	<section class="wok pd">
		<div class="container">
			<h1 class="wok__title section__title"><span>WOK</span></h1>
			<div class="products-carousel">
				<div class="products-carousel__body">
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" min="0" value="0">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">
											Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
					<div>
						<div class="carousel-item">
							<div class="carousel-item__image">
								<img src="img/wok/wok-1.jpg" alt="wok-1">
							</div>
							<div class="carousel-item__inner">
								<h3 class="carousel-item__name">Сет №10</h3>
								<p class="carousel-item__info">72 шт. | 1560 г. | 1700 Kkal</p>
								<div class="carousel-item__wrap">
									<div class="carousel-item__quantity">
										<span>Добавить:</span>
										<input type="number" value="0" min="0" class="quantity-input">
									</div>
									<div class="carousel-item__sause-price">140<sup>руб.</sup></div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__select select">
										<div class="select__input">
											Китайский соус
											<div class="select__input-btn">&#8595;</div>
										</div>
										<div class="select__dropdown">
											<ul class="select__dropdown-list">
												<li class="select__dropdown-item">1 item</li>
												<li class="select__dropdown-item">2 item</li>
												<li class="select__dropdown-item">3 item</li>
												<li class="select__dropdown-item">4 item</li>
												<li class="select__dropdown-item">5 item</li>
											</ul>
										</div>

									</div>
								</div>
								<div class="carousel-item__wrap">
									<div class="carousel-item__price">2 050 <sup>руб.</sup></div>
									<div class="carousel-item__count counter">
										<div class="counter__btn counter__btn-minus">-</div>
										<div class="counter__number">1</div>
										<div class="counter__btn counter__btn-plus">+</div>
									</div>
								</div>
								<button class="btn carousel-item__btn">В корзину</button>
							</div>
						</div>
					</div>
				</div>
				<div class="products-carousel__control carousel-control">
					<div class="carousel-control__btn carousel-control__btn-prev"><span></span></div>
					<div class="carousel-control__dots"></div>
					<div class="carousel-control__btn carousel-control__btn-next"><span></span></div>
				</div>
			</div>
		</div>
	</section>

	<section class="promotion pd">
		<div class="container">
			<div class="promotion__tabs">
				<div class="promotion__tabs-buttons">
					<div class="tab-btn active">Новинки</div>
					<div class="tab-btn">Акции</div>
				</div>
				<div class="promotion__tabs-bodies">

				</div>
			</div>
		</div>
	</section>
	<button class="btn carousel-item__btn">В корзину</button>


	<script src="js/libs.min.js"></script>
	<script src="js/main.js"></script>
	<script src="js/shop.js"></script>
</body>

</html>