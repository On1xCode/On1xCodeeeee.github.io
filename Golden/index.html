<!DOCTYPE html>
<html lang="ru">

<head>
	<meta charset="utf-8">
	<title>Golden</title>
	<meta name="description" content="Startup HTML template OptimizedHTML 5">
	<meta name="viewport" content="width=device-width">
	<link rel="icon" href="img/favicon.png">
	<meta property="og:image" content="img/@1x/preview.jpg">

	<script src="https://kit.fontawesome.com/74ca5f1a9a.js" crossorigin="anonymous"></script>

	<link rel="stylesheet" href="css/styles.min.css">
</head>

<body>
	<header class="header" id="home">
		<div class="container">
			<div class="header-top flex-wrapper jc-sb ai-c">
				<a href="#" class="header-top-logo">
					Golden
				</a>
				<nav class="header-top-nav">
					<ul>
						<li><a href="#home" class="menu-link">Home</a></li>
						<li><a href="#services" class="menu-link">Services</a></li>
						<li><a href="#portfolio" class="menu-link">Portfolio</a></li>
						<li><a href="#about" class="menu-link">About</a></li>
						<li><a href="#team" class="menu-link">Team</a></li>
						<li><a href="#contact" class="menu-link">Contact</a></li>
					</ul>
				</nav>
				<div class="header-top-hamburger">
					<i class="fas fa-hamburger"></i>
				</div>
			</div>
			<div class="header-content">
				<h2 class="header-content-subtitle">Welcome To Our Studio!</h2>
				<h1 class="header-content-title">Itâ€™s nice to Meet you</h1>
				<a href="#" class="btn header-btn">Tell Me More</a>
			</div>
		</div>
		<div class="header-adp-menu">
			<nav>
				<ul>
					<li><a href="#home" class="menu-link">Home</a></li>
					<li><a href="#services" class="menu-link">Services</a></li>
					<li><a href="#portfolio" class="menu-link">Portfolio</a></li>
					<li><a href="#about" class="menu-link">About</a></li>
					<li><a href="#team" class="menu-link">Team</a></li>
					<li><a href="#contact" class="menu-link">Contact</a></li>
				</ul>
			</nav>
		</div>
	</header>

	<section class="services" id="services">
		<div class="container">
			<h3 class="section-title services-title">Services</h3>
			<p class="section-subtitle services-subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="flex-wrapper jc-sb ai-c">
				<div class="services-item">
					<div class="services-item-img" style="background-image: url(img/service1.png)"></div>
					<h3 class="services-item-title">E-Commerce</h3>
					<p class="services-item-text">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services-item">
					<div class="services-item-img" style="background-image: url(img/service2.png)"></div>
					<h3 class="services-item-title">Responsive Web</h3>
					<p class="services-item-text">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services-item">
					<div class="services-item-img" style="background-image: url(img/service3.png)"></div>
					<h3 class="services-item-title">Web Security</h3>
					<p class="services-item-text">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
			</div>
		</div>
	</section>

	<section class="portfolio" id="portfolio">
		<div class="container">
			<h3 class="section-title portfolio-title">Our portfolio</h3>
			<p class="section-subtitle portfolio-subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="flex-wrapper jc-c fx-wrap">
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf1.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Ebony & Ivory
						</h3>
						<p class="portfolio-item-text">Branding</p>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf2.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Smart Stationary
						</h3>
						<p class="portfolio-item-text">Print Design</p>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf3.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Clever Poster
						</h3>
						<p class="portfolio-item-text">Print Design</p>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf4.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Vinyl Record
						</h3>
						<p class="portfolio-item-text">Product Mock-Up</p>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf5.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Treehouse Template</h3>
						<p class="portfolio-item-text">Web Design</p>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="portfolio-item-img" style="background-image: url(img/portf6.jpg)"></div>
					<div class="colored">
						<h3 class="portfolio-item-title">Burned Logo
												</h3>
						<p class="portfolio-item-text">Branding</p>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section class="about" id="about">
		<div class="container">
			<h3 class="section-title about-title">Our portfolio</h3>
			<p class="section-subtitle about-subtitle">Proin iaculis purus consequat sem cure.</p>

			<div class="grid-wrapper">
				<div class="about-text story-text-1">
					<h3 class="about-text-title">JULY 2010 <br>Our Humble Beginnings</h3>
					<p class="about-text-descr">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
				</div>
				<div class="about-img story-img-1" style="background-image: url(img/story1.jpg);"></div>
				<div class="about-text story-text-2">
					<h3 class="about-text-title">JANUARY 2011 <br> Facing Startup Battles</h3>
					<p class="about-text-descr">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
				</div>
				<div class="about-img story-img-2" style="background-image: url(img/story2.jpg);"></div>
				<div class="about-text story-text-3">
					<h3 class="about-text-title">DECEMBER 2012 <br>Enter The Dark Days</h3>
					<p class="about-text-descr">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
				</div>
				<div class="about-img story-img-3" style="background-image: url(img/story3.jpg);"></div>
				<div class="about-text story-text-4">
					<h3 class="about-text-title">FEBRUARY 2014 <br>Our Triumph</h3>
					<p class="about-text-descr">Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
				</div>
				<div class="about-img story-img-4" style="background-image: url(img/story4.jpg);"></div>
				<div class="about-img continue">Our <br> Story <br> Continues <br> ...</div>
			</div>
		</div>
	</section>

	<section class="team" id="team">
		<div class="container">
			<h3 class="section-title team-title">our amazing team</h3>
			<p class="section-subtitle team-subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="flex-wrapper jc-c fx-wrap">
				<div class="team-member">
					<div class="team-member-img"></div>
					<h3 class="team-member-name">Kimberly Thompson</h3>
					<p class="team-member-profession">Marketer</p>
					<span class="team-member-social">
						<a href="#" class="twitter"><i class="fab fa-twitter"></i></a>
						<a href="#" class="facebook"><i class="fab fa-facebook-f"></i></a>
						<a href="#" class="pinterest"><i class="fab fa-pinterest-p"></i></a>
					</span>
				</div>
				<div class="team-member">
					<div class="team-member-img"></div>
					<h3 class="team-member-name">Rico Massimo</h3>
					<p class="team-member-profession">Coder</p>
					<span class="team-member-social">
						<a href="#" class="twitter"><i class="fab fa-twitter"></i></a>
						<a href="#" class="facebook"><i class="fab fa-facebook-f"></i></a>
						<a href="#" class="pinterest"><i class="fab fa-google-plus-g"></i></a>
					</span>
				</div>
				<div class="team-member">
					<div class="team-member-img"></div>
					<h3 class="team-member-name">Uku Mason</h3>
					<p class="team-member-profession">Graphic Designer</p>
					<span class="team-member-social">
						<a href="#" class="facebook"><i class="fab fa-facebook-f"></i></a>
						<a href="#" class="pinterest"><i class="fab fa-pinterest-p"></i></a>
					</span>
				</div>
			</div>
			<p class="team-text">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit
			aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
		</div>
	</section>

	<section class="partners">
		<div class="container">
			<div class="flex-wrapper jc-sa ai-c fx-wrap">
				<a href="#"><img src="img/partner1.png" alt="Envato"></a>
				<a href="#"><img src="img/partner2.png" alt="WordPress"></a>
				<a href="#"><img src="img/partner3.png" alt="AETuts+"></a>
				<a href="#"><img src="img/partner4.png" alt="Microlancer"></a>
			</div>
		</div>
	</section>

	<section class="contact" id="contact">
		<div class="container">
			<h3 class="section-title contact-title">Contact us</h3>
			<p class="section-subtitle contact-subtitle">Proin iaculis purus consequat sem cure.</p>
			<form action="#" class="contact-form">
				<div class="flex-wrapper jc-c ai-c">
					<div class="column">
						<input type="text" placeholder="YOUR NAME*">
						<input type="text" placeholder="YOUR E-MAIL*">
						<input type="text" placeholder="YOUR SUBJECT">
					</div>
					<div class="column">
						<textarea name="message" placeholder="YOUR MESSAGE*"></textarea>
					</div>
				</div>
				<button type="button" class="btn">Send Message</button>
			</form>
		</div>
	</section>

	<footer class="footer">
		<div class="container">
			<p>Â© Copyright 2014 FreebiesXpress.com</p>
			<span class="footer-social">
				<a href="#" class="twitter"><i class="fab fa-twitter"></i></a>
				<a href="#" class="facebook"><i class="fab fa-facebook-f"></i></a>
				<a href="#" class="pinterest"><i class="fab fa-pinterest-p"></i></a>
				<a href="#" class="pinterest"><i class="fab fa-google-plus-g"></i></a>
			</span>
		</div>
	</footer>




	<script src="js/jquery.min.js"></script>
	<script src="js/scripts.min.js"></script>
</body>
</html>
